DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/compute_units/src/compute_units.cpp

```diff
@@ -0,0 +1,110 @@
+//==============================================================
+// Copyright Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+// =============================================================
+#include <CL/sycl.hpp>
```

Comment:
Probably in 2020:
```suggestion
#include <SYCL/sycl.hpp>
```

Reply:
This will break compile

Reply:
Yes I think we need to keep CL/sycl.hpp, as in: https://github.com/intel/llvm/blob/sycl/sycl/include/CL/sycl.hpp